src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -44,8 +44,17 @@
 
             doc.querySelector(relatedCollapseSelector).classList.remove('ibexa-collapse--hidden');
             afterChangeGroup();
+            toggleAddGroupTriggerBtnState();
         },
     });
+    const toggleAddGroupTriggerBtnState = () => {
+        const addGroupBtns = doc.querySelectorAll('.ibexa-content-type-edit__add-field-definitions-group [data-related-collapse-selector]');
+        const areEveryAddGroupBtnsDisabled = [...addGroupBtns].every((btn) =>
```

Comment:
```suggestion
        const areAllAddGroupBtnsDisabled = [...addGroupBtns].every((btn) =>
```
is best. (also as a sidenote - if you use "every", all other words should be singular, so in that case "isEveryAddGroupBtnDisabled"), but still - best one is one in suggestion ;)